projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -86,12 +86,15 @@ public static function jetpack_run_scheduled_update( $plugins = array() ) {
 	 * @return bool
 	 */
 	public static function jetpack_allowlist_scheduled_plugins( $update, $item ) {
-		// TODO: Check if we're in a scheduled update request from Jetpack_Autoupdates.
-		$schedules = get_option( 'jetpack_update_schedules', array() );
 
-		foreach ( $schedules as $plugins ) {
-			if ( in_array( $item->slug, $plugins, true ) ) {
-				return true;
+		if ( isset( $item->is_scheduled_updates ) && $item->is_scheduled_updates ) {
```

Comment:
Rather than changing the shape of Core's update object for this, how would this look like with constants defined, along the lines of `JETPACK_PLUGIN_AUTOUPDATE`?

For context, I'd also love to avoid the change in `jetpack_run_scheduled_update()` further up, in case you have an idea for a good alternative (in a new diff).

Reply:
> Rather than changing the shape of Core's update object for this, how would this look like with constants defined, along the lines of JETPACK_PLUGIN_AUTOUPDATE?

Yes, not changing the shape of Core's update object is a good point, I've modified the code and it checks the `SCHEDULED_AUTOUPDATE` constant now.